.github/workflows/ci.yml

```diff
@@ -129,7 +129,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: [ net6.0, net7.0 ]
+        version: [ net6.0 ]
```

Comment:
I think we should add `net8.0` here. Not sure why it is missing.

Reply:
The Docker setup [failed](https://github.com/open-telemetry/opentelemetry-dotnet/actions/runs/10480814956/job/29029133563?pr=5795) for net8.0 after adding this. Not sure if it ever worked for .NET 8.

Reply:
I tried to fix it with `venv`: https://stackoverflow.com/questions/75602063/pip-install-r-requirements-txt-is-failing-this-environment-is-externally-mana

But it's not quite trivial.

I think it's better to add in another PR. I'll revert adding net8.0 for this step and keep this PR simple.

Reply:
could you please open an issue to track?

Reply:
Opened an issue: #5799.